.github/workflows/broken-links-checker.yml

```diff
@@ -0,0 +1,19 @@
+name: Broken links checker
+on:
+  push:
+    paths:
+      - '.github/workflows/broken-links-checker.yml'
+  schedule:
+    - cron: '*/28 6 * * *' # Run once daily at arbitrary time (6:28 AM UTC)
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-node@v2-beta
+        with:
+          node-version: '12'
+      - run: npm install broken-link-checker -g
+      - name: Run broken-link-checker on neptune.ai
+        run: blc https://neptune.ai -ro --requests 5 --host-requests 5 --exclude 'https://*.linkedin.com/**' --get
```

Comment:
I was wondering whether link checking neptune.ai it should be in the docs CI. 
I think in our particular setup, it does make sense but perhaps we'll drop it (move it) in the future.